### PR TITLE
V0.2.3

### PR DESCRIPTION
**Pull request by bollig**
*Description of changes:*

- cleaned up queue names (now ondemand, spot, efa and gpu)
- multiple compute resources to on-demand and spot queues
- added timeout on cloud9 after 90 min of inactivity
- added missing placement_groups on queues
- updated to pcluster v2.10.3

### CODE REVIEW

File: scripts/sc20workshop.ini [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[aws]
```
Comment (sean-smith) [inline scripts/sc20workshop.ini:1]: What's the point of this file?

File: inherit.yaml [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
             EnableBudget: false
             CreateServiceLinkedRoles: false
+            Cloud9Owner: !Sub 'arn:aws:sts::${AWS::AccountId}:assumed-role/TeamRole/MasterKey'
+            ConfigS3URI: 'https://raw.githubusercontent.com/aws-samples/no-tears-cluster/v0.2.3/scripts/sc20workshop.ini'
+Outputs:
+  NoTearsCloud9:
+    Value:
+      Fn::GetAtt:
+          - NoTearsHPC
+          - Outputs.ResearchWorkspaceURL
```
Comment (sean-smith) [inline inherit.yaml:18]: This is cool!

File: pcluster/pcluster_stack.py [hunk 3]
```diff
@@ -165,7 +169,23 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
...
+            "Properties.OwnerArn",
+            {
+                "Fn::If": [
+                    c9owner_condition.logical_id,
+                    cdk.Aws.NO_VALUE,
+                    c9_owner.value_as_string
+                ]
+            }
+        )
+        cloud9_instance.node.default_child.add_override(
```
Comment (sean-smith) [inline pcluster/pcluster_stack.py:185]: why is this included?

**Review by sean-smith — APPROVED** [review]

File: pcluster/pcluster_stack.py (hunk 3 above)
Comment (bollig) [reply pcluster/pcluster_stack.py:185]: This allows us to override the Cloud9 owner to assign an AssumedRole (useful for AWS Organizations and delegated roles).

File: scripts/sc20workshop.ini (hunk 1 above)
Comment (bollig) [reply scripts/sc20workshop.ini:1]: This is an alternative config based on https://sc20.hpcworkshops.com/. Useful for tutorials and accelerating onboarding  
/ pre-provisioning clusters for workshop attendees.
